components/review/stages/reviewInfoStage.tsx

```diff
@@ -170,13 +104,35 @@ export const ReviewInfoStage: React.FC<Props> = ({ scores, reviewId }) => {
       id: reviewId,
     }).then((result) => {
       if (result.data) {
+        const email = result.data.applicationsById.email;
```

Comment:
For all these variables, let's make DTO and fill it the object fields with these variables instead. Will make code easier to read and more typed.